src/C#/pablopickaxo-fibonacci.cs

```diff
@@ -0,0 +1,26 @@
+// Simple solution for printing fibonacci sequence!
+
+using System;  
+
+  public class FibonacciExample  
```

Comment:
Rename the class to match the name in filename, or vice versa

Reply:
Will do!